gen/definitions/wireless_profile.yaml

```diff
@@ -1,43 +1,39 @@
 ---
 name: Wireless Profile
-rest_endpoint: /dna/intent/api/v2/wireless/profile
+rest_endpoint: /intent/api/v1/wirelessProfiles
 id_from_query_path: response
+id_from_query_path_attribute: id
+import_no_id: true
 get_from_all: true
-no_data_source: true
-put_no_id: true
-id_from_query_path_attribute: instanceUuid
 res_description:
   'This resource creates a wireless network profile. To associate a wireless network profile with a site, use 
   the `catalystcenter_associate_site_to_network_profile` resource.'
 doc_category: Wireless
+test_tags: [CC2376]
```

Comment:
Do we need this? I think we are testing with 2.3.7.6+ anyway.

Reply:
removed